var/spack/repos/builtin/packages/mpich/fj.3.4.patch

```diff
@@ -0,0 +1,422 @@
+++++ spack-src/maint/patches/optional/confdb/f77.patch	2021-03-03 16:29:40.498102556 +0900
++@@ -0,0 +1,18 @@
+++--- libtool.m4.org	2021-03-03 16:21:54.800529915 +0900
++++++ libtool.m4	2021-03-03 16:29:34.959297925 +0900
+++@@ -6291,6 +6291,7 @@
+++ ## the running order or otherwise move them around unless you know exactly
+++ ## what you are doing...
+++ if test -n "$compiler"; then
++++  _LT_SYS_HIDDEN_LIBDEPS($1)
+++   _LT_COMPILER_NO_RTTI($1)
+++   _LT_COMPILER_PIC($1)
+++   _LT_COMPILER_C_O($1)
+++@@ -7829,6 +7830,7 @@
+++     ## There is no encapsulation within the following macros, do not change
+++     ## the running order or otherwise move them around unless you know exactly
+++     ## what you are doing...
++++    _LT_SYS_HIDDEN_LIBDEPS($1)
+++     _LT_COMPILER_PIC($1)
+++     _LT_COMPILER_C_O($1)
+++     _LT_COMPILER_FILE_LOCKS($1)
++diff -rNu spack-src.org/maint/patches/optional/confdb/fujitsu.patch spack-src/maint/patches/optional/confdb/fujitsu.patch
++--- spack-src.org/maint/patches/optional/confdb/fujitsu.patch	1970-01-01 09:00:00.000000000 +0900
+++++ spack-src/maint/patches/optional/confdb/fujitsu.patch	2021-03-03 16:34:43.539413435 +0900
++@@ -0,0 +1,42 @@
+++--- libtool.m4.org	2021-03-03 16:21:54.800529915 +0900
++++++ libtool.m4	2021-03-03 16:34:02.020877909 +0900
+++@@ -4789,6 +4789,11 @@
+++         # ARM LLVM C/C++/Fortran
+++         _LT_TAGVAR(lt_prog_compiler_wl, $1)='-Wl,'
+++         ;;
++++      fcc* | FCC* | frt*)
++++        # Fujitsu C/C++/Fortran
++++        _LT_TAGVAR(lt_prog_compiler_wl, $1)='-Wl,'
++++       _LT_TAGVAR(lt_prog_compiler_pic, $1)='-KPIC'
++++        ;;
+++       *)
+++ 	case `$CC -V 2>&1 | sed 5q` in
+++ 	*Sun\ Ceres\ Fortran* | *Sun*Fortran*\ [[1-7]].* | *Sun*Fortran*\ 8.[[0-3]]*)
+++@@ -7814,6 +7819,13 @@
+++ 	  esac
+++ 	fi
+++         ;;
++++      linux* | k*bsd*-gnu | kopensolaris*-gnu | gnu*)
++++        case $cc_basename in
++++          fcc* | FCC* | frt*)
++++            output_verbose_link_cmd='$CC $CFLAGS -v conftest.$objext 2>&1 | $GREP -v "^Configured with:" | $GREP "\-L"'
++++          ;;
++++        esac
++++        ;;
+++     esac
+++     AC_MSG_RESULT([$enable_shared])
+++ 
+++@@ -7952,6 +7964,13 @@
+++ 	  esac
+++ 	fi
+++         ;;
++++      linux* | k*bsd*-gnu | kopensolaris*-gnu | gnu*)
++++        case $cc_basename in
++++          fcc* | FCC* | frt*)
++++            output_verbose_link_cmd='$CC $CFLAGS -v conftest.$objext 2>&1 | $GREP -v "^Configured with:" | $GREP "\-L"'
++++          ;;
++++        esac
++++        ;;
+++     esac
+++     AC_MSG_RESULT([$enable_shared])
+++ 
+diff -rNu spack-src/maint/patches/optional/confdb/fujitsu.patch spack-src-fj/maint/patches/optional/confdb/fujitsu.patch
+--- spack-src/maint/patches/optional/confdb/fujitsu.patch	1970-01-01 09:00:00.000000000 +0900
++++ spack-src-fj/maint/patches/optional/confdb/fujitsu.patch	2021-03-08 12:36:57.654873969 +0900
```

Comment:
We would definitely like to have this libtool patch upstream. Is there a way we can test fujitsu compiler on our systems via container or otherwise?